api/api/clubs/views.py

```diff
@@ -14,3 +15,22 @@ class UpdateClubsView(UpdateAPIView):
     permission_classes = (IsPositionHolderOrAdmin,)
     queryset = Club.objects.all()
     serializer_class = ClubSerializer
+
+
+class CreateClubRegistrationRequestView(CreateAPIView):
+    queryset = ClubRegistrationRequest.objects.all()
+    serializer_class = ClubRegistrationRequestSerializer
+
+
+class UpdateClubRegistrationRequestView(UpdateAPIView):
+    permission_classes = (IsCoreMemberOrAdmin,)
+    serializer_class = ClubUpdateRegistrationRequestSerializer
+    # pk_2 is registration request id that is requested to be updated
+    lookup_url_kwarg = "pk_2"
+
+    def get_queryset(self):
+        # pk_1 is the club id wrt to given registration request
+        queryset = ClubRegistrationRequest.objects.filter(club=self.kwargs['pk_1'])
+        if queryset:
+            return queryset
+        return ClubRegistrationRequest.objects.none()
```

Comment:
Does these lines achieve anything?

We could simply return `ClubRegistrationRequest.objects.filter(club=self.kwargs['pk_1'])`